tests/integration-test/src/tests/e2e.rs

```diff
@@ -341,67 +328,61 @@ fn test_program_execution_with_global_variables() {
         &LoadType::Image,
         TC_PASS_IMAGE_LOC,
         TC_PASS_FILE_LOC,
-    )
-    .unwrap();
+    );
 
-    uuids.push(uuid);
+    uuids.push(prog_id.unwrap());
```

Comment:
nit: it's probably just ids now but that's alrigt, ultimately all this needs to be deduplicated anyways

Reply:
Done, was already on my list to do.